exercises/complex-numbers/description.md

```diff
@@ -1,29 +1,85 @@
 # Description
 
-A complex number is a number in the form `a + b * i` where `a` and `b` are real and `i` satisfies `i^2 = -1`.
+A **complex number** is expressed in the form:
 
-`a` is called the real part and `b` is called the imaginary part of `z`.
-The conjugate of the number `a + b * i` is the number `a - b * i`.
-The absolute value of a complex number `z = a + b * i` is a real number `|z| = sqrt(a^2 + b^2)`. The square of the absolute value `|z|^2` is the result of multiplication of `z` by its complex conjugate.
+```math
+z = a + b * i
+```
 
-The sum/difference of two complex numbers involves adding/subtracting their real and imaginary parts separately:
-`(a + i * b) + (c + i * d) = (a + c) + (b + d) * i`,
-`(a + i * b) - (c + i * d) = (a - c) + (b - d) * i`.
+where:
 
-Multiplication result is by definition
-`(a + i * b) * (c + i * d) = (a * c - b * d) + (b * c + a * d) * i`.
+- `a` is the **real part** (a real number),
 
-The reciprocal of a non-zero complex number is
-`1 / (a + i * b) = a/(a^2 + b^2) - b/(a^2 + b^2) * i`.
+- `b` is the **imaginary part** (also a real number), and
 
-Dividing a complex number `a + i * b` by another `c + i * d` gives:
-`(a + i * b) / (c + i * d) = (a * c + b * d)/(c^2 + d^2) + (b * c - a * d)/(c^2 + d^2) * i`.
+- `i` is the imaginary unit satisfying `i^2 = -1`.
 
-Raising e to a complex exponent can be expressed as `e^(a + i * b) = e^a * e^(i * b)`, the last term of which is given by Euler's formula `e^(i * b) = cos(b) + i * sin(b)`.
+## Key Properties
 
-Implement the following operations:
+1. **Conjugate**: The conjugate of the complex number `z = a + b * i` is given by:
```

Comment:
Instead of making this a list, try:

```markdown
### Conjugate

[...]

```

For example below this line:


-------------------------------

## Key Properties

### Conjugate

The conjugate of the complex number `z = a + b * i` is given by:

```math
z̅ = a - b * i
```

### Absolute Value

The absolute value (or modulus) of `z` is defined as:

```math
|z| = sqrt(a^2 + b^2)
```

Reply:
@jagdish-15 Go with SleeplessByte's suggestion, disregard mine.